admin/data/sql/migrations/144.do.create-pin-table.sql

```diff
@@ -0,0 +1,14 @@
+-- create a pin table
+DROP TABLE IF EXISTS [mtc_admin].[pin];
+create table [mtc_admin].[pin] (
+  [id] [int] IDENTITY(1,1) NOT NULL,
+  [createdAt] [datetimeoffset](3) NOT NULL
+    CONSTRAINT[DF_pin_created_at] DEFAULT CURRENT_TIMESTAMP,
```

Comment:
we should use `GETUTCDATE()` for consistency

Reply:
done